docs/contracts/inheritance.rst

```diff
@@ -250,6 +252,20 @@ overridden when used with multiple inheritance:
     // No explicit override required
     contract D is B, C {}
 
+More formally, it is not required to override a function (directly or
+indirectly) inherited from multiple bases if there is a base contract
+that is part of all override paths for the signature, and (1) that
+base implements the function and no paths from the current contract
+to the base mentions a function with that signature or (2) that base
+does not implement the function and there is at most one mention of
+the function in all paths from the current contract to that base
+and that mention (if available) implements it.
```

Comment:
I don't see this last part in the code.

Reply:
That part may actually be missing in the current version (in the case there's only one function left we'd need to verify that it's implemented).